### DISCUSSION TITLE
Dataset Overview with No-code

### DISCUSSION THREAD

**Discussion by pierrebrunelle** [awslabs/fortuna#18] [Show and tell]
Hi

I took the Time-series Regression with EnbPI and imported here: https://app.noteable.io/f/9a874c9e-e12f-4e57-b9a6-5ee44a527c57/Time-series-regression-with-EnbPI.ipynb

I made some nice visualizations to show the shape of the data

![image](https://user-images.githubusercontent.com/70675979/224403151-fcc5e780-9d46-4d37-82f0-b361629bb0b5.png)

But I'm trying to run the entire notebook and getting stuck at the Online Feedback section, where the samples have a different size, so it errored out.

**Comment by gianlucadetommaso** [comment]
Nice visuals! Fixed now, thanks for spotting this.
